examples/next/docs/angular/basic-example/package.json

```diff
@@ -18,33 +18,33 @@
     "test": "node spec/support/jasmine.config.js"
   },
   "dependencies": {
-    "@angular/animations": "latest",
-    "@angular/common": "latest",
-    "@angular/compiler": "latest",
-    "@angular/core": "latest",
-    "@angular/forms": "latest",
-    "@angular/platform-browser": "latest",
-    "@angular/platform-browser-dynamic": "latest",
-    "@angular/router": "latest",
-    "@handsontable/angular": "12.0.1",
+    "@angular/animations": "^16.0.0",
+    "@angular/common": "^16.0.0",
+    "@angular/compiler": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@angular/forms": "^16.0.0",
+    "@angular/platform-browser": "^16.0.0",
+    "@angular/platform-browser-dynamic": "^16.0.0",
+    "@angular/router": "^16.0.0",
```

Comment:
For `/next/` examples, shouldn't the "latest" tag be used here?

Reply:
Yup https://github.com/handsontable/handsontable/pull/10396/commits/9cab13b56a5540b0a740bb2d7604c4345bfd5898